cmake/Dependencies.cmake

```diff
@@ -50,7 +50,7 @@ if(USE_CUDA)
     if(NOT CAFFE2_USE_NVRTC)
       caffe2_update_option(USE_NVRTC OFF)
     endif()
-    list(APPEND Caffe2_CUDA_DEPENDENCY_LIBS caffe2::curand caffe2::cufft caffe2::cublas)
+    list(APPEND Caffe2_CUDA_DEPENDENCY_LIBS torch::curand torch::cufft torch::cublas)
```

Comment:
Again, why are we renaming those here?

Reply:
Caffe2 is gone, take the chance to rename.